run.sh

```diff
@@ -71,10 +86,12 @@ solana-genesis \
   --bootstrap-storage-pubkey "$dataDir"/leader-storage-account-keypair.json \
   --ledger "$ledgerDir" \
   --dev
+tar jcfS "$ledgerDir/genesis.tar.bz2" -C "$ledgerDir" genesis.bin rocksdb
```

Comment:
Ack!  I just noticed https://github.com/solana-labs/solana/issues/6953.   Can you please fix #6953 before merging this PR and then we can delete this line

Reply:
I see!

Reply:
(actually I don't have objections to fixing #6953 after this PR merges either, no need to block really)

Reply:
thx; I'll go merge this first. I know #6953 will be small, but I want this rather soon. :) I'll remove this like in the upcoming #6953 PR.